cli/src/cmd/error.rs

```diff
@@ -60,6 +82,14 @@ impl From<&str> for CommandError {
     }
 }
 
+impl From<String> for CommandError {
```

Comment:
Maybe you can unify `From<&str>` and `From<String>` by implementing for `T: Into<String>`.

Reply:
This doesn't work because some of the other errors e.g. `IoError` already implement `Into<String>` which causes conflicting implementations with `derive_more::From`